### ISSUE TITLE
Should the sequence diagram also show that the "Driving App" receives the context change event?  A requester gets the event as well as other subscribers, correct?

### ISSUE THREAD

**Issue by isaacvetter** [HL7/fhircast-docs#444]
Should the sequence diagram also show that the "Driving App" receives the context change event?  A requester gets the event as well as other subscribers, correct?

Also, "Drving App" should be "Driving App" unless intentionally made shorter.

_Originally posted by @EricOnFHIR in https://github.com/HL7/fhircast-docs/pull/440#discussion_r1011982201_

**Comment by bvdh** [comment]
Yes typically it should. In what diagram do you want to make the change?
